v0/destinations/splitio/data/EventConfig.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "destKey": "eventTypeId",
+    "sourceKeys": [
+      "eventTypeId",
+      "properties.eventTypeId",
```

Comment:
priority should be event > type (no need to take from properties/traits) 
regex for this according to document is [a-zA-Z0-9][-_\.a-zA-Z0-9]{0,62} please check this and then only send event otherwise throw error.

Reply:
eliminated properties and traits part. Kept the name part, because when we process page or screen call, event field is not present but name field is present. Later on in transform.js we are adding `viewed name/ page or screen` like segment.